### DISCUSSION TITLE
The most useful analytics

### DISCUSSION THREAD

**Discussion by AlexSim93** [AlexSim93/pull-request-analytics-action#4] [Polls]
I need to understand which features are considered the most useful by the community. The results may change the default parameters or shift my focus to new features. Feel free to share your ideas about any desirable features!

**Comment by usamakzafar** [comment]
Still trying to run this, getting http errors after waiting 12-13 minutes-long batch request. As a feedback, I would suggest creating 2 different steps, the first being all the batch requests and the second being creating the report.

That way, once the batch request is done, we don't have to run it every time due to a failure in creating the report

**Reply by AlexSim93** [reply]
Caching data to make action faster is in my to do list. But it takes time to implement it. But I'm still considering different option for results. Right now you can get collection, markdown or new issue created. Just today I have posted [Q&A](https://github.com/AlexSim93/pull-request-analytics-action/discussions/12) to understand user's needs.
